.idea/runConfigurations/Artemis__Server__Athene_.xml

```diff
@@ -10,4 +10,4 @@
       <option name="Make" enabled="true" />
     </method>
   </configuration>
-</component>
+</component>
```

Comment:
Thank you for removing your change, but this `.xml` should end on a newline and not on a space if I am not mistaken.
Have a look at [this](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log).

Reply:
```suggestion
</component>

```
IntelliJ doesn't let me do that 🤔 . Will try from here

Reply:
worked out well, Thanks!